src/trialscope.ts

```diff
@@ -42,9 +178,19 @@ export class TrialScopeQuery {
   zipCode?: string = null;
   travelRadius?: number = null;
   phase = 'any';
-  recruitmentStatus = 'all';
+  recruitmentStatus: string | string[] | null = null;
   after?: string = null;
   first = 30;
+  /**
+   * The fields that should be returned within the individual trial object.
+   */
+  trialFields = [
+    'nctId', 'title', 'officialTitle', 'conditions', 'keywords', 'gender',
+    'description', 'detailedDescription', 'criteria', 'sponsor',
+    'overallContactName', 'overallContactPhone', 'overallContactEmail',
+    'overallOfficialName', 'overallStatus', 'armGroups', 'phase', 'minimumAge',
+    'studyType', 'countries', 'maximumAge'
+  ];
```

Comment:
Should minimum and maximum age be added to the TrialScopeTrial object

Reply:
I don't think they are currently being used for anything, so I guess they don't need to be there.